src/components/image/index.tsx

```diff
@@ -93,15 +120,25 @@ export function Image({
         onSave,
         onError
       );
+
       if (newObjectUrl) {
         setImgObjectUrl(newObjectUrl);
       }
     }
   };
+
+  let imageComponent = null;
+
+  if (src) {
+    /* eslint-disable-next-line react/jsx-props-no-spreading */
```

Comment:
I've seen this one quite a few times already, I propose we just get rid of this ESLint rule :P

Reply:
It's _typically_ a good rule, except in edge cases where we explicitly want to spread.